crates/driver/src/infra/config/file/mod.rs

```diff
@@ -654,3 +659,7 @@ fn default_order_priority_strategies() -> Vec<OrderPriorityStrategy> {
 fn default_max_order_age() -> Option<Duration> {
     Some(Duration::from_secs(300))
 }
+
+fn default_settle_queue_size() -> usize {
+    3
+}
```

Comment:
Why is this equal to 3? Is there some logic behind it? Would be nice to add a comment how we estimated this?

Reply:
Added a comment explaining it.